.env.development

```diff
@@ -1,3 +1,4 @@
-NEXT_PUBLIC_API_BASE_URI=http://localhost:8000
-NEXT_PUBLIC_SITE_BASE_URI=http://localhost:3000
-NEXT_PUBLIC_MATOMO_WEBSITE_ID=38
+NEXT_PUBLIC_API_BASE_URI=https://flathub.org/api/v2
+NEXT_PUBLIC_SITE_BASE_URI=https://flathub.org
```

Comment:
Are these URI changes correct for development?

Reply:
They are not, they snuck in from a system, I usually don't have a backend on. Will fix asap.

Reply:
Hopefully good now, sorry!